Translated Descriptions

DahliaSR opened the pull request:
Closes #2

DahliaSR committed:
Translated Description

- part of #1

DahliaSR committed:
Translated Short Description

Closes #2

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -1,68 +1,105 @@
+        »Warum kannst Du sie nicht benutzen!?«, fragte Sofia beinahe hysterisch.<br>
+        »Ich habe sie schon vergessen! Willst Du, dass ich für alle Ewigkeit aus meinem
+        eigenen Haus ausgeschlossen bin? Lass Dir gefälligst was Einfacheres einfallen,
+        sodass ich mir nicht diesen zufälligen Kauderwelsch merken muss.«, keiffte Sophia.<br>
```
Sofia (instead Sophia), but I recommend to end the speak without mentioning her again anyway. It's done before, it is clear Sofia is still speaking. Your choice of a verb isn't helping too :)

DahliaSR replied:
:white_check_mark:

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -107,16 +144,14 @@
-    The earliest known description of the grille cipher comes from the Italian mathematician, Girolamo Cardano in 1550.
+    <strong>Wozu das gut ist:</strong><br>
+    Hier erfährst Du, wie man mit 2-D-Arrays arbeiten kann.<br>
+    Außerdem lernst Du etwas die Fleißnersche Schablone, ein Verschlüsselungsverfahren aus
```
lieber
..lernst Du etwas über...
oder
...lernst Du wie die Fleißnersche Schablone funktioniert...
oder
..lernst Du etwas die Fleißnersche Schablone kennen...

DahliaSR replied:
:white_check_mark:

DahliaSR committed:
Fixex according to Review

https://github.com/CheckIO-to-German/Cipher-Map/pull/3#pullrequestreview-10665416

DahliaSR commented:
@GiForce hier fehlt noch dein "Approval"
